Improve error message mentioning to use --verbose by specifying where to place it in the command line

**Is your feature request related to a problem? Please describe.**

When a user has a connection issue or mistyped a Catalog like:
```
 jbang run invalid@invalid
```

the error message is:

```
[jbang] [ERROR] Unknown catalog 'invalid'
[jbang] Run with --verbose for more details
```

if you type:
```
 jbang run invalid@invalid --verbose
```

the output is the same.

The command to have more information is:
```
 jbang  --verbose run invalid@invalid
```

**Describe the solution you'd like**

In case, the --verbose must be placed before the run, I would like to have the error message mentioning it.
Not sure if it applies to all cases or not.

**Describe alternatives you've considered**

ideally if the message can contain the exact command to use based on the command received that could be nice. (can we hit security issues like if some passwords/token are provided? through environment variable? I guess we would have access only to the command with replaced values as I suspect the sheel environment is responsible to replace them.

**Additional context**

Some users were blocked because there was the unknown catalog without more messages. Using the --verbose at the right place revealed a proxy configuration issue.

--- a/src/main/java/dev/jbang/util/Util.java
+++ b/src/main/java/dev/jbang/util/Util.java
@@ -467,7 +467,7 @@ public class Util {
 			if (msg != null) {
 				infoMsg(e.getMessage());
 			}
-			infoMsg("Run with --verbose for more details");
+			infoMsg("Run with --verbose for more details. The --verbose must be placed before the jbang command. I.e. jbang --verbose run [...]");
 		}
 	}
 

